### DISCUSSION TITLE
Can I visualize ip ranges and available ips in gui? (Under prefixes)

### DISCUSSION THREAD

**Discussion by sliddjur** [netbox-community/netbox#14797] [Q&A]
When I go to prefixes and press the IP addresses tab, is there an option to show: 
- ip ranges
- available ips
as well as the used and documented ips? 
and each available ip had a green color link to add that specific ip.

in my opinion this would be a nice feature to be able to quickly see available ips within a given prefix. also an option to show the ip-ranges inside a prefix would be nice so that I dont create duplicates. 

wasnt this a feature on older netbox versions to atleast show available ips?

**Comment by abhi1693** [comment]
Are you referring to this feature? I grabbed it from the demo server running v3.7.0

![image](https://github.com/netbox-community/netbox/assets/5083532/ee790f47-2a82-4ad6-88c3-08672da62499)

**Reply by sliddjur** [reply]
Yes. I have 3.7.0, but I cant see that under prefix tab.
I have also made an ip-range from 10.6.69.200/24 to 10.6.69.254 (for dhcp reservation), I'd like that to be visible under IPs aswell.

This is what it looks like for me.
![image](https://github.com/netbox-community/netbox/assets/10002057/1b36eb0b-b8ab-442a-b2f6-ed4cb42591f1)

**Reply by sliddjur** [reply]
I realised that my view doesnt show IPs, because I have enable sorting.
After I pressed the little red X mark on the column header, it will now show "Available IPs".

**Comment by candlerb** [comment]
Seeing both child IPs and child ranges merged into the same view has been requested before. Otherwise, you might manually assign an IP address without realising that it is actually within a range, which is reserved for some special purpose.

I'm not sure where the canonical issue/discussions is but I found #6934, #10495.  I thought at some point I'd proposed some specific logic for the merged view, but I can't find it now.

**Comment by sliddjur** [comment]
@candlerb yeah I see you commented in #6934 .
Even if I mark ip-range as 100% utilized, and at the same time `ENFORCE_GLOBAL_UNIQUE = True` I will still be able to create that IP in netbox. 
My intent is to somehow reserve IPs for a DHCP scope, and make sure that noone is reserving IPs within that range. Right now I see no way of clearly visualizing this in the IP Address overview inside of a prefix.
